log10/__main__.py

```diff
@@ -53,5 +61,8 @@ def feedback_task():
 feedback_task.add_command(list_feedback_task, "list")
 feedback_task.add_command(get_feedback_task, "get")
 
+cli.add_command(autofeedback)
+autofeedback.add_command(get_autofeedback, "get")
+
```

Comment:
I'm wondering if we want to make this to a separate cli group or keep it with the `feedback`, where the `auto_feedback_icl` is now. so user do 
`log10 feedback auto_feedback`

Reply:
also want to update the cli doc: https://github.com/log10-io/docs.log10.io/blob/main/pages/cli_docs.mdx